### PR TITLE
OCPBUGS#1516: Updated reference to IBM Cloud VPC account to IBM Cloud…

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.10+

Issue:
This PR addresses [OCPBUGS#1516](https://issues.redhat.com/browse/OCPBUGS-1516).

Link to docs preview:

- [Preparing to install on IBM Cloud VPC](https://50595--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.html)
- [Configuring an IBM Cloud account](https://50595--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-account.html)
- [Installing a cluster on IBM Cloud VPC with customizations](https://50595--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.html)
- [Installing a cluster on IBM Cloud VPC with network customizations](https://50595--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.html)
- [Exporting the IBM Cloud VPC API key](https://50595--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.html#installation-ibm-cloud-export-variables_installing-ibm-cloud-network-customizations)

### CODE REVIEW

**Comment by mjpytlak** [comment]
Given the nature of this update QE review is not required. See [OCPBUGS#1516](https://issues.redhat.com/browse/OCPBUGS-1516) for details.

File: installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ include::snippets/technology-preview.adoc[]
 [id="requirements-for-installing-ocp-on-ibm-cloud"]
 == Requirements for installing {product-title} on IBM Cloud VPC
 
-Before installing {product-title} on IBM Cloud VPC, you must create a service account and configure an IBM Cloud VPC account. See xref:../../installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc#installing-ibm-cloud-account[Configuring an IBM Cloud VPC account] for details about creating an account, enabling API services, configuring DNS, IBM Cloud VPC account limits, and supported IBM Cloud VPC regions.
+Before installing {product-title} on IBM Cloud VPC, you must create a service account and configure an . See xref:../../installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc#installing-ibm-cloud-account[Configuring an IBM Cloud account] for details about creating an account, enabling API services, configuring DNS, IBM Cloud account limits, and supported IBM Cloud VPC regions.
```
Comment (michaelryanpeter) [inline installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc:24]:
```suggestion
Before installing {product-title} on IBM Cloud VPC, you must create a service account and configure an <IBM Cloud accoutn?>. See xref:../../installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc#installing-ibm-cloud-account[Configuring an IBM Cloud account] for details about creating an account, enabling API services, configuring DNS, IBM Cloud account limits, and supported IBM Cloud VPC regions.
```

**Review by michaelryanpeter — COMMENTED** [review]
Just a small typo that lead to an incomplete sentence.

**Pull request merged**